move-vm-backend/src/warehouse.rs

```diff
@@ -1,7 +1,4 @@
-use crate::{
-    storage::Storage,
-    //    SubstrateAPI,
```

Comment:
Could you do me a favour and undo a change in this file? Since my next PR will take care of this :grinning:

Reply:
@Rqnsom of course! What do you want me to change exactly?

Reply:
Done